test.tf

```diff
@@ -35,6 +35,8 @@ resource "aws_ami" "awsAmiEncrypted" {
     device_name = "dev-name"
     encrypted = "false"
   }
+  
+  
 }
```

Comment:
**Tenable.cs remediation**
 
**[HIGH]: Ensure MFA Delete is enable on S3 buckets**
 
**Vulnerability**
Description:

Once MFA Delete is enabled on your sensitive and classified S3 bucket it requires the user to have two forms of authentication.

Rationale:

Adding MFA delete to an S3 bucket, requires additional authentication when you change the version state of your bucket or you delete and object version adding another layer of security in the event your security credentials are compromised or unauthorized access is granted.
 
**Remediation**
Perform the steps below to enable MFA delete on an S3 bucket.

Note:
-You cannot enable MFA Delete using the AWS Management Console. You must use the AWS CLI or API.
-You must use your 'root' account to enable MFA Delete on S3 buckets.

**From Command line:**

1. Run the s3api put-bucket-versioning command

```
aws s3api put-bucket-versioning --profile my-root-profile --bucket Bucket_Name --versioning-configuration Status=Enabled,MFADelete=Enabled --mfa “arn:aws:iam::aws_account_id:mfa/root-account-mfa-device passcode”
```.

source: tenablecs-